src/main/webapp/app/exercises/quiz/manage/drag-and-drop-question/drag-and-drop-question-edit.component.html

```diff
@@ -117,27 +199,40 @@ <h3 class="mb-0"><span class="badge bg-warning align-text-top">DnD</span></h3>
                 ></jhi-markdown-editor>
             </div>
             <div class="markupEditorArea" *ngIf="reEvaluationInProgress">
-                <div class="question-content">
-                    <jhi-markdown-editor
-                        #markdownEditor
-                        [markdown]="questionEditorText"
-                        [showPreviewButton]="false"
-                        [enableResize]="true"
-                        [domainCommands]="dragAndDropQuestionDomainCommands"
-                        (markdownChange)="changesInMarkdown()"
-                        (textWithDomainCommandsFound)="domainCommandsFound($event)"
-                        class="h-auto"
-                    ></jhi-markdown-editor>
-                    <span class="input-group-btn" style="vertical-align: top">
-                        <button class="btn btn-outline-secondary btn-lg" type="button" (click)="resetQuestionText()">
-                            <fa-icon [icon]="faUndo"></fa-icon>
-                        </button>
-                    </span>
+                <div class="question-content row">
+                    <div class="col-10 col-lg-11">
+                        <!-- TODO: Fix file upload    -->
+                        <jhi-markdown-editor
+                            #markdownEditor
+                            [markdown]="questionEditorText"
+                            [showPreviewButton]="false"
+                            [enableResize]="true"
+                            [domainCommands]="dragAndDropQuestionDomainCommands"
+                            (markdownChange)="changesInMarkdown()"
+                            (textWithDomainCommandsFound)="domainCommandsFound($event)"
+                            [enableFileUpload]="false"
+                            class="h-auto"
+                        ></jhi-markdown-editor>
+                    </div>
+                    <div class="col-2 col-lg-1">
+                        <span class="input-group-btn" style="vertical-align: top">
+                            <button
+                                class="btn btn-outline-secondary"
+                                type="button"
+                                (click)="resetQuestionText()"
+                                [ngbTooltip]="'artemisApp.quizExercise.re-evaluate.resetQuestionText' | artemisTranslate"
+                                style="margin-top: 40px"
```

Comment:
Use bootstrap util classes if possible, as otherwise we have inconsistent spacings

Reply:
I removed this style to keep the button aligns with other element.